composer.json

```diff
@@ -73,7 +73,7 @@
         "ext-ctype": "*",
         "lib-pcre": "*",
         "yiisoft/yii2-composer": "~2.0.4",
-        "ezyang/htmlpurifier": "^4.6",
+        "ezyang/htmlpurifier": ">=4.6",
```

Comment:
This is an unbound version constraint and should be avoided.
Which version would you need? 

`^4.6` would install `4.17` btw: https://semver.madewithlove.com/?package=ezyang%2Fhtmlpurifier&constraint=%5E4.6

Reply:
I need 4.7 to get installed (and a coffee obviously)